### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by gulzainali98** [HymEric/VSR-DUF-Reimplement#14]
Were you able to achieve paper results?

**Comment by HymEric** [comment]
Sorry for replying so late.
Actually not. Part of the reason is the loss of training data. The author of the paper can't provide the dataset.
